charts/argo-rollouts/templates/controller/notifcations-configmap.yaml

```diff
@@ -16,3 +16,7 @@ data:
   {{- with .Values.notifications.triggers }}
     {{- toYaml . | nindent 2 }}
   {{- end }}
+  {{- with .Values.notifications.subscriptions }}
+  subscriptions:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Comment:
nits; please add new blank line in the end

Reply:
Done!